plugin/storage/cassandra/schema/create.sh

```diff
@@ -30,9 +31,19 @@ if [[ "$MODE" == "" ]]; then
     usage "missing MODE parameter"
 elif [[ "$MODE" == "prod" ]]; then
     if [[ "$DATACENTER" == "" ]]; then usage "missing DATACENTER parameter for prod mode"; fi
+    replication_factor=${REPLICATION_FACTOR:-3}
```

Comment:
You should be able to just set the env var `REPLICATION_FACTOR=3` on the container running this script.  I don't think we should change the default value here.

Reply:
This is sort of the can of worms I was poking at in the issue, and understand the hesitancy to change to 3. I am not a Cassandra expert by any stretch of imagination, but was informed internally that for quorum, 3 should be the default vs. 2.

Also, if you are running `create.sh` manually or within self baked images, that would work fine, but the issue is that the `docker.sh` (lines 42-46) is where the script is invoked, which is provided by the `jaeger-cassandra-schema` image. From what I understand, folks using the Jaeger Operator Helm Chart(like ourselves), we do not have a way to push in the replication factor via `values.yaml`.

For Example:
```
...
        cassandraCreateSchema:
          enabled: true
          datacenter: us-west2,europe-west4
          mode: "prod"
          image: my-custom-images/jaeger-cassandra-schema-bootstrap:latest
...
```

Or am I overthinking this?

Reference link to `docker.sh`:
https://github.com/jaegertracing/jaeger/blob/44a6d37edeeabb669dc1df61a2099bed09c1bb9a/plugin/storage/cassandra/schema/docker.sh#L45

Reply:
There is no "right" value for replication factor, there are pros and cons for different settings. I agree that for backwards compatibility it's best not to change the current default.

Regarding the config schema above, it sounds like it needs a change in the operator.

Reply:
@yurishkuro I agree on both fronts. I think making the config more flexible at the operator level is ultimately the right solution.